### PR TITLE
Use Hazelcast version in the protocol definitions

### PR DESCRIPTION
**Pull request by mdumandag**
We were using the separate protocol version in the since values
of the protocol definitions, but that was causing a quite bit
of confusion.

We have decided to stick to the Hazelcast versions to make it simpler.

This PR updates the auto-generated test files with the changes
done on the protocol side and updates the since versions used in
the `LongRegister`.

protocol PR: https://github.com/hazelcast/hazelcast-client-protocol/pull/444

### CODE REVIEW

**Review by srknzl — APPROVED** [review]

**Comment by mdumandag** [comment]
run-lab-run

File: hazelcast/src/test/java/com/hazelcast/client/protocol/compatibility/ClientCompatibilityTest_5_3.java
Comment (srknzl) [inline hazelcast/src/test/java/com/hazelcast/client/protocol/compatibility/ClientCompatibilityTest_5_3.java:1]: This is interesting, should not we have 5.3 test already?

Comment (srknzl) [reply hazelcast/src/test/java/com/hazelcast/client/protocol/compatibility/ClientCompatibilityTest_5_3.java:1]: We have ClientCompatibilityTest_2_6 already in master

**Review by srknzl — CHANGES_REQUESTED** [review]
looks good except a comment

**Pull request closed without merging**